{"ctfname": "0x41414141", "challenges": [{"name": "pyjail", "solver": ["Rezkon"], "solve_date": 1611560674, "category": "misc"}]}